layouts/default.vue

```diff
@@ -59,6 +86,11 @@
     </v-main>
     <v-footer app>
       <span>&copy; 2020</span>
+      <h5 :class="`text-md-h6`" class="ml-3" style="font-weight:normal">
+        <a :href="baseURL" class="no-underline underline-on-hover black--text" hover="underline">
```

Comment:
You can actually use `<NuxtLink to="/">` instead of `<a>`; reserve `<a>` for external links.
Try that?